test/client/sender-app/startSpec.js

```diff
@@ -1,13 +1,15 @@
 'use strict';
 
 describe('Start Controller', function() {
-  var scope, ctrl; // using this in our tests
+  var scope, ctrl, playerMessenger, playerUser; // using this in our tests
 
   // mock app
   beforeEach(function() {
     angular.mock.module('app');
 
-    inject(function($rootScope, $controller) {
+    inject(function($rootScope, $controller, $state, _playerMessenger_, _playerUser_) {
```

Comment:
Officially, at this point, no... but plans to add more state tests once I figure that out